en/controllers/components/authentication.rst

```diff
@@ -109,6 +109,8 @@ keys.
 - ``passwordHasher`` Password hasher class; Defaults to ``Default``.
 - The ``scope`` and ``contain`` options have been deprecated as of 3.1. Use
   a custom finder instead to modify the query to fetch a user record.
+- The ``userFields`` option has been deprecated as of 3.1. Use ``select()`` in 
+  your custom finder.
```

Comment:
It's referred to as 'a custom finder' above in the paragraph about 'scope' and 'contain' so I thought it would be more consistent to refer to 'custom finder' so that it was talking about the same thing.

As the query is the subject of the finder method, I would think calling `select()` would make sense.